src/canvas.jl

```diff
@@ -34,7 +34,7 @@ function points!(
 ) where {T<:UserColorType}
     length(X) == length(Y) == length(color) ||
         throw(DimensionMismatch("X, Y, and color must be the same length"))
-    for i in 1:length(X)
+    for i in eachindex(X)
```

Comment:
Because here we don't have `@inbounds` so it's quite "safe"

Reply:
I know, juste being picky on good practices :)